backend/src/test/java/gov/cdc/usds/simplereport/idp/repository/OktaRepositoryTest.java

```diff
@@ -0,0 +1,61 @@
+package gov.cdc.usds.simplereport.idp.repository;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Disabled;
+
+import java.util.Optional;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.idp.repository.OktaRepository;
+import gov.cdc.usds.simplereport.config.authorization.OrganizationRole;
+import gov.cdc.usds.simplereport.config.authorization.OrganizationRoleClaims;
+import gov.cdc.usds.simplereport.service.model.IdentityAttributes;
+
+class OktaRepositoryTest {
```

Comment:
I think it might be helpful to add a javadoc saying what these tests are and why we aren't running them.

I'd also love to get them running, at least optionally, but that's a separate question.

Reply:
looking to get them running in v2